CONTRIBUTING.md

```diff
@@ -115,6 +115,35 @@ To ensure consistency and quality throughout the source code, before being accep
 **Please note**:
 We will be happy to help you out in meeting these requirements. Don't be afraid to share your code in a pull request even if it doesn't meet the requirements yet.
 
+#### Code Structure
+
+We try to follow the "Newspaper code structure", see https://kentcdodds.com/blog/newspaper-code-structure. (Yes, we are aware that the author has archived that blog post because he doesn't follow that principle anymore, but we still think it's a good fit for us.)
+
+We put all non-function properties on top, so we want to order them with public properties first, then more important private properties, then less important private properties.
```

Comment:
```suggestion
We try to follow the "Newspaper code structure", which in short means putting all non-function properties on top; and ordering them with public properties first, then more important private properties, then less important private properties.
```

Reply:
People can always Google stuff. If we think using this is good, there is no need to bring up excuses. Let's keep it simple.

Reply:
Ok 👍